README.md

```diff
@@ -24,9 +24,9 @@ bootstrap({
 
  * Creates and starts the server listening for connections.
  * `@param {Object}` options
- * `@return {Object} bootstrap` interface object.
+ * `@return {Promise.<bootstrap, Error>}` a promise that returns bootstrap interface object if resolved, Error if rejected.
```

Comment:
I don't follow, promise notation isn't very well documented - I went with how Medium do it - https://github.com/jsdoc3/jsdoc/issues/509#issuecomment-29186993. Could you give an example if we do it differently?